### ISSUE TITLE
Lacking documentation

### ISSUE THREAD

**Issue by DamienCassou** [narendraj9/hledger-mode#35]
Hi,

thank you for your package. I think the project lacks some documentation. For example:

1. the animated GIF in the README looks great but the features it demonstrates are not described anywhere. Would it be possible to have a list of the commands used in the demo?
2. I don't see any list of features that the package supports nor how to use any feature.

**Comment by narendraj9** [comment]
I totally agree. When I started working on this, I was building it mostly for myself. Some features are not used at all (not even by me). It's possible they do not work anymore.

**Comment by simonmichael** [comment]
Hi @narendraj9. I use ledger-mode because of some useful features and it is a while since I tried hledger-mode. I see there were some commits last year. Are you looking for more contributors ? A useful addition to the readme could be a comparison with ledger-mode, reasons to use one or the other, and maybe more discussion of next steps if any ?

**Comment by simonmichael** [comment]
Also, FWIW the example emacs config is probably full of useful tips, but at first glance it scared me - could be worth also showing a minimal one for getting started ?

**Comment by simonmichael** [comment]
> could be worth also showing a minimal one

Oops, it's already there - my bad!

**Comment by narendraj9** [comment]
> Hi @narendraj9. I use ledger-mode because of some useful features and it is a while since I tried hledger-mode. I see there were some commits last year. Are you looking for more contributors ? A useful addition to the readme could be a comparison with ledger-mode, reasons to use one or the other, and maybe more discussion of next steps if any ?

Hey @simonmichael , I am not actively working on the project. If you would like to contribute or become a maintainer, that will be great :)

**Comment by simonmichael** [comment]
Thanks for the update @narendraj9. I use ledger-mode myself and don't have time to help much with this, but I'll put the word out.

**Comment by Askath** [comment]
@narendraj9 Hey, I love this package and would like to contribute. Would it be best to just open PRs?

**Comment by narendraj9** [comment]
Hey @Askath, thank you. Yes, please feel free to raise PRs or start a discussion in an issue if you think the change is significant.
